### PR TITLE
Improve sample-app

### PR DESCRIPTION
**Pull request by MegaGM**

### CODE REVIEW

File: sample/dockerwait.js [hunk 1]
```diff
@@ -13,11 +13,10 @@ var bootCheckInterval = Number(process.env.SOCKETCLUSTER_BOOT_CHECK_INTERVAL) ||
 var bootTimeout = Number(process.env.SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT) || 10000;
 var bootStartTime = Date.now();
 
-var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} ` +
-`before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
+var errorMessage = `Failed to locate the master controller file at path ${masterControllerPath} before SOCKETCLUSTER_CONTROLLER_BOOT_TIMEOUT`;
```
Comment (MegaGM) [inline sample/dockerwait.js]: There are plenty of >80chars lines all over the codebase, so it doesn't make much sense to split error messages.

File: sample/dockerwait.js [hunk 2]
```diff
@@ -13,11 +13,10 @@ var bootCheckInterval = Number(process.env.SOCKETCLUSTER_BOOT_CHECK_INTERVAL) ||
...
 
 waitForFile(masterControllerPath, bootCheckInterval, bootStartTime, bootTimeout, errorMessage)
-.catch((err) => {
-  console.error('> Boot error: ' + err.message);
-  process.exit(1);
-});
+  .catch((err) => {
```
Comment (MegaGM) [inline sample/dockerwait.js]: Should I exclude any changes made by autobeautifier from comments or is it fine to include them?
Or, maybe, it's better extract such changes into separate commits?

Comment (jondubois) [reply sample/dockerwait.js]: Yes, better exclude them because it's against the current convention. Right now there is no indentation if the line starts with a dot.

**Comment by jondubois** [comment]
@MegaGM Thanks :)
